tests/unit_node/sqlite_test.ts

```diff
@@ -73,6 +73,27 @@ Deno.test("[node/sqlite] StatementSync read bigints are supported", () => {
   assertEquals(stmt.get(), { key: 1n, __proto__: null });
 });
 
+Deno.test("[node/sqlite] createSession and changesets", () => {
```

Comment:
Do we have to create these from scratch? Can't we use tests from Node's suite?

Reply:
Yep we should also enable the Node.js tests.

Reply:
I guess not right now because the sqlite API is not fully on par with Node.js for the tests to pass.